HRHN/View/VC/SettingViewController.swift

```diff
@@ -0,0 +1,170 @@
+                SettingItem(text: "홈페이지", secondaryText: nil, type: .defaultItem, imageName: "globe", link: "https://hrhn.notion.site/f7ecd6dca58046b298ad8debfbcc762e"),
+                SettingItem(text: "오픈소스 라이선스", secondaryText: nil, type: .defaultItem, imageName: "chevron.left.forwardslash.chevron.right", link: "https://hrhn.notion.site/2dd3252ad190433392c58f77e975cb18")
+            ], header: "SUPPORT")
+        ]
+    }
+}
+
+final class SettingViewController: UIViewController {
+    
+    // MARK: - Properties
+    
+    let list = SettingSection.generateData()
+    
+    private lazy var titleView = UIView()
+    private lazy var titleLabel: UILabel = {
+        $0.text = "설정"
+        $0.textColor = .label
+        $0.font = .systemFont(ofSize: 25, weight: .bold)
+        $0.numberOfLines = 0
+        return $0
+    }(UILabel())
+    
+    private lazy var tableView: UITableView = { [weak self] in
+        $0.register(SettingCell.self, forCellReuseIdentifier: SettingCell.identifier)
+        $0.delegate = self
+        $0.dataSource = self
+        $0.alwaysBounceVertical = false
+        $0.showsVerticalScrollIndicator = false
+        return $0
+    }(UITableView())
+    
+    // MARK: - LifeCycle
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+        setUI()
+        setNavigationBar()
+    }
+    
+}
+
+// MARK: - Functions
+extension SettingViewController {
+    
+    @objc func backButtonDidTap(_ sender: UIButton) {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+}
+
+// MARK: - UI Functions
+extension SettingViewController {
+    private func setUI(){
+        view.backgroundColor = .background
+        view.addSubviews(titleView, tableView)
+        titleView.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide)
+            $0.horizontalEdges.equalToSuperview()
+            $0.height.equalTo(66)
+        }
+        titleView.addSubviews(titleLabel)
+        titleLabel.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(20)
+            $0.leading.equalToSuperview().offset(20)
+        }
+        tableView.snp.makeConstraints {
+            $0.top.equalTo(titleView.snp.bottom)
+            $0.horizontalEdges.equalToSuperview()
+            $0.bottom.equalTo(view.safeAreaLayoutGuide)
+        }
+    }
+    
+    private func setNavigationBar() {
+        let leftBarButton: UIBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "chevron.left"), style: .plain, target: self, action: #selector(backButtonDidTap))
+        navigationItem.leftBarButtonItem = leftBarButton
+    }
```

Comment:
```suggestion
    private func setNavigationBar() {
        navigationController?.navigationBar.topItem?.title = ""
    }
```
내비게이션바 'Back' 글씨 없애는거 저는 위 방법으로 했는데 chevron 굵기가 달라서 통일하는게 좋을거 같아요! 변경 부탁드려도 될까요?
leftBarButton을 커스텀으로 주면 뒤로가기 제스처가 막히더라고요

Reply:
아하 그런건 바로 
반영했습니다!
6ebba8b3fbdf86762082706f6bca044a0dabbee2